gnucash/gnome/assistant-stock-transaction.cpp

```diff
@@ -514,7 +515,17 @@ refresh_page_transaction_type (GtkWidget *widget, gpointer user_data)
     if (!info->txn_types)
         return;
 
-    info->txn_type = info->txn_types->at (type_idx);
+    try
+    {
+        info->txn_type = info->txn_types->at (type_idx);
+    }
+    catch (const std::out_of_range& e)
```

Comment:
The exception is not used in this block and would lead to a warning about unused variable in `-Wall`. How about `catch(const std::out_of_range&)`?